config/locales/titles/en.yml

```diff
@@ -63,7 +63,14 @@ en:
     revoke_consent: Revoke Consent
     rules_of_use: Rules of Use
     sign_up:
+      completion_consent_expired: It’s been a year since you gave us consent to share your information
+      completion_first_sign_in: You've created an account with ${app_name}
+      completion_ial2: You've verified your identity with ${app_name}
+      completion_new_attributes: ${sp} is requested new information
+      completion_new_sp: You are now signing in for the first time
+
```

Comment:
nit: remove these blank lines
```suggestion
```

Reply:
Yeah, these are here for when I go back and need to figure out which of the old translations I still need to keep around and which I can get rid of. I'll clean them up when I get to that.